Create ui-c8y-1023-14-43-prevent-additional-requests-when-dpg-is-in-the-real-time-mode.md (#4130)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-14-43-prevent-additional-requests-when-dpg-is-in-the-real-time-mode.md

---
date: ""

> title: prevent additional requests when the data points graph is in real time mode

Reviewer: ```suggestion
title: Prevented additional requests of the Data point graph widget in real-time mode
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65548
version: 1023.14.43
---

> When the data points graph widget operates in real time mode, it was making unnecessary additional requests to fetch data, which could impact performance and increase server load.

Reviewer: ```suggestion
When the "Data point graph" widget operated in real-time mode, it made unnecessary requests to fetch data, which could impact performance and increase server load. This issue has been fixed.
```
